### ISSUE TITLE
Unwanted sound when moving to desktop using the "Move To" menu

### ISSUE THREAD

**Issue by veriton** [Eun/MoveToDesktop#32]
This is a nice utility, thanks :)

When I pick a Desktop from the "Move To" menu it makes a sound though. I don't know if this is something to do with my system (Win 10 Anniversary Update x86-64) as a quick scan through the main.cpp code doesn't seem to be calling anything to make a sound. If it is a deliberate choice I would prefer an option in MoveToDesktop.ini to disable it..

Note: when I use the WIN+ALT+LEFT/RIGHT shortcut key it doesn't make a sound.

Simon

**Comment by Eun** [comment]
Uh oh, never heard this before, can you record the sound? Is it a windows sound?
MoveToDesktop should not play any sounds...

**Comment by lukaszadamski** [comment]
I also get sound when I move any window. It is a standard sound signal.

**Comment by veriton** [comment]
Yes, it's a standard Windows notification/alert for me - perhaps as a result of some call failing? Note the sound doesn't happen with the keyboard short-cut so perhaps that may narrow down the suspects in the code.

**Comment by lukaszadamski** [comment]
Surprisingly this sounds also happen to me with keyboard shortcut. but... not all the times... when I move chrome it happens, when word or notepad++ not :) 

It doesn't bother me, just letting you know.

**Comment by Eun** [comment]
If you set Sound Scheme to `No Sounds` does it still play?
![untitled](https://cloud.githubusercontent.com/assets/796084/19850708/413c7dd2-9f5a-11e6-926d-34c71bac46f8.png)

**Comment by dasebasto** [comment]
Here is what happens on my computer:

- **Moving a Chrome window using hotkey** -> sound
- Moving a Chrome window using context menu -> no sound
- **Moving a Explorer.exe window using hotkey** -> only in some windows/circumstances, the sound plays (when I release the Alt key, not before. this is different from the Chrome case, where the sound is played immediately, when the window disappears)
- Moving a Explorer.exe window using context menu -> no sound
- Moving Notepad++, Android Studio (and some other apps)  window (and  using hotkey or context menu-> no sound


The sound is a default sound from Windows called "Default Beep". As expected, it does not play if Windows Sound Scheme is set to "No sounds".

Good luck with the investigation :)

**Comment by linusfr** [comment]
Hey, first of all, thank you for this nice addition to the desktop environment. 
I know this is already a year old, but did you work on this sound problem? I have it as well. But only with Chrome, everything else doesnt make a sound moving it. When is the "Default Beep" sound as well, and it does not play if i disable windows sounds. 

I am on the Windows 10 Education edtition. The version ist 1709.
